### ISSUE TITLE
Typo in weewx.conf

### ISSUE THREAD

**Issue by Pogs2004** [chaunceygardiner/weewx-xtide#1]
Thanks for this great extension and installation instructions, I was struggling to get XTide working on my RPi 5 Ubuntu 24.04 LTS / WeeWX 5.1 box for the Forecast extension. I followed your documentation to the letter but still failed when I tried the test to run tide which invoked "Command 'tide' not found, but can be installed with: sudo apt install xtide". I had to run it as a system call ./ then it worked perfectly. XTide works fine for both your extension and the Forecast extension.

**Comment by chaunceygardiner** [comment]
Thanks for the report.  I’m struggling to find the line with the typo.  In the README?  Which line?

**Comment by Pogs2004** [comment]
My apologies, that's the wrong title! There was a typo in my weewx.conf that had locoation instead of location. I have no idea how it got there as I only noticed it in the syslog when it said a location must be specified. I checked your python script for the weewx.conf entries and there was no typo, so I assume it must have been me. The only issue I had was executing line 7 of the docs: run /home/<username>/xtide-2.15.5/tide -l "Palo Alto Yacht Harbor", I had to use ./tide -l as it would not run with just tide.
